django/db/models/query.py

```diff
@@ -1263,12 +1262,10 @@ def columns(self):
         columns = self.query.get_columns()
         # Adjust any column names which don't match field names
         for (query_name, model_name) in self.translations.items():
-            try:
+            with suppress(ValueError):
+                # Ignore translations for nonexistent column names
```

Comment:
Not really. I considered this when finding the `try`-blocks, and where to put the comment, and ended up doing it that way. I'll move them one line up :)